osgp/platform/osgp-domain-distributionautomation/src/main/java/org/opensmartgridplatform/domain/da/measurements/Measurement.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ */
+package org.opensmartgridplatform.domain.da.measurements;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Measurement {
+    private List<MeasurementElement<?>> measurementElements = new ArrayList<>();
+
+    public Measurement(final List<MeasurementElement<?>> measurementElements) {
+        this.measurementElements = new ArrayList<>(measurementElements);
+    }
+
+    public List<MeasurementElement<?>> getMeasurementElements() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1452)